### ISSUE TITLE
Does not work with new routing of Contao 4.13

### ISSUE THREAD

**Issue by webaffin** [alnv/catalog-manager#344]
I tried new routing feature of Contao 4.13 with option 
`legacy_routing: false`
 in config.yml.
This broke my site, because catalog manager still uses old hooks 
`getPageIdFromUrl`
and/or
`getRootPageFromUrl`
Are there plans to get rid of the old hooks in future versions?

**Comment by alnv** [comment]
Should fixed in v1.32.1

**Comment by webaffin** [comment]
Ja, kann bestätigen, dass es nun anstandslos läuft mit Catalog Manager. Dankeschön <3
